src/MDOP.jl

```diff
@@ -4,16 +4,150 @@ using Distances
 
 export beta_statistic
 export MDOP
+export estimate_maximum_delay
 
 
 """
-    MDOP(....)
-method by [^Nichkawde2013]
+    MDOP(s::Vector; kwargs...) → Y, τ_vals, ts_vals, FNNs [,βS]
```

Comment:
What does `MDOP` stand for?

Reply:
haha, you are right. I'll explain the abbrevation